src/components/Overlays/OverlayHeaderUsername.tsx

```diff
@@ -0,0 +1,33 @@
+import { ServiceCredentialBindingObj } from '@/api/cf/cloudfoundry-types';
+import { Username } from '@/components/UserAccount/Username';
+import { Tag } from '@/components/uswds/Tag';
+
+export function OverlayHeaderUsername({
+  header,
+  serviceAccount,
+  username,
+}: {
+  header: string;
+  serviceAccount?: ServiceCredentialBindingObj | undefined | null;
+  username: string;
+}) {
+  return (
+    <>
+      <h4
+        className="margin-top-0 margin-bottom-7 text-uppercase text-light underline-base-light text-underline"
```

Comment:
Could we bump the font size down on this slightly? (It looks to be ~16px in the prototype; I was aiming for 14px in the design.)